Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1/conversion_generated.go

```diff
@@ -1564,6 +1564,11 @@ func convert_api_PodSpec_To_v1_PodSpec(in *api.PodSpec, out *PodSpec, s conversi
 	} else {
 		out.ImagePullSecrets = nil
 	}
+
+	// Carry conversion
+	out.DeprecatedServiceAccount = in.ServiceAccountName
+	out.DeprecatedHost = in.NodeName
```

Comment:
On Jul 6, 2015, at 6:40 PM, Jordan Liggitt <notifications@github.com> wrote:

In
Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1/conversion_generated.go
<https://github.com/openshift/origin/pull/3592#discussion_r33990776>:

> @@ -1564,6 +1564,11 @@ func convert_api_PodSpec_To_v1_PodSpec(in *api.PodSpec, out *PodSpec, s conversi
>  	} else {
>  		out.ImagePullSecrets = nil
>  	}
> +
> +	// Carry conversion
> +	out.DeprecatedServiceAccount = in.ServiceAccountName
> +	out.DeprecatedHost = in.NodeName

This is only here because I'm assuming we have to round-trip back to the
deprecated field... @smarterclayton <https://github.com/smarterclayton>, is
that true?

Yes

Also, I hate modifying a generated file but I couldn't find a cleaner place
to put it

As long as it's in its own commit.  Ultimately we'll run unit tests and
maybe even generators against our version, which would make this less yucky.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3592/files#r33990776>.